app/javascript/models/assessment.js

```diff
@@ -26,4 +27,12 @@ export default class Assessment extends BaseModel {
     return get(first(this.variants), 'format_id');
   }
 
+  @computed get preview_html() {
+    return map(this.variants, 'preview_html').join('');
```

Comment:
Is this joining the previews from all variants?  if so we probably don’t want that, because there could be 2000 of them — if we allow to cycle through them later, we may need some other functionality to help us with that.

Reply:
going to merge, we can address later.